src/components/JosekiVariationFilter/JosekiVariationFilter.tsx

```diff
@@ -79,16 +75,9 @@ export function JosekiVariationFilter(props: JosekiVariationFilterProps) {
                 console.log("Contributors GET failed:", r);
             });
 
-        fetch(props.source_list_url, {
-            mode: "cors",
-            headers: props.oje_headers,
-        })
-            .then((res) => res.json())
+        get(props.source_list_url)
             .then((body) => {
-                // This can possibly be changed to !== undefined or != null,
-                // But I don't want to touch this right now -BPJ
-                // eslint-disable-next-line eqeqeq
-                if (body.sources != undefined) {
+                if (body.sources != null) {
```

Comment:
The comment in here was telling us that the sentry logs shows that sometimes `body.sources` was uyndefined ... it would have been some message like `trying to read thingy, undefined`.

I have no idea why I used `!=` and not `!==`  ... but I guess since null == undefined, this is all good.